Releases.md

```diff
@@ -4,6 +4,7 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 
 | Version                  | Feature level                                                                                                            | First available in OS                                             | Redistributable                                                                             |
 | ------------------------ | ------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| 1.15.2 | [DML_FEATURE_LEVEL_6_4](https://learn.microsoft.com/windows/ai/directml/dml-feature-level-history#dml_feature_level_6_4) | TBD                                                               | [Microsoft.AI.DirectML.1.15.2](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.15.2) |
```

Comment:
Presuming we'll add `(#directml-1152)` later.

Reply:
No, there is no release notes to add.